cli/cmd/root.go

```diff
@@ -222,6 +311,22 @@ func newCli() (*cli.CLI, error) {
 			return nil, fmt.Errorf("failed to create VMClarity API client: %w", err)
 		}
 
+		if exportCICDResults && scanResultID == "" {
+			i, err := cicdinitiator.NewInitiator(client, config, scanConfigID, scanConfigName, input, inputType)
+			if err != nil {
+				return nil, fmt.Errorf("failed to create VMClarity initiator: %w", err)
+			}
+			var scanID string
+			scanID, scanResultID, err = i.InitResults(context.TODO())
+			if err != nil {
+				return nil, fmt.Errorf("failed to init scan result: %w", err)
+			}
```

Comment:
seems like all the logic to get the `scanID` and `scanResultID` can be extracted to a function, the `initiator` is not used after that, right?

Reply:
Yes, I put it into a function.